### PR TITLE
added Microsoft 365 groups

### PR DESCRIPTION
**Pull request by PramodBalusu**

### CODE REVIEW

File: SharePoint/SharePointOnline/change-site-address.md [hunk 1]
```diff
@@ -164,6 +164,9 @@ Site customizations and embedded code that refer to URLs might need to be fixed
 The OneDrive sync app will automatically detect and seamlessly transfer syncing to the new site URL after the site address has been changed. Users don't need to sign in again or take any other action. (Version 17.3.6943.0625 or later of the sync app required.) 
 If a user updates a file while the site address is being changed, they'll see a message that file uploads are pending during the change. 
 
+**Microsoft 365 groups**<br>
+The email address of the group will not be renamed. The group name is updated only if site name is updated during the rename process.
```
Comment (JohanFreelancer9) [inline SharePoint/SharePointOnline/change-site-address.md:168]:
```suggestion
The email address of the group will not be renamed. The group name is updated only if the site name is updated during the rename process. 
```

**Review by JohanFreelancer9 — CHANGES_REQUESTED** [review]
Copy review - @PramodBalusu, kindly commit the minor suggested change. Many thanks.

**Comment by PramodBalusu** [comment]
Please merge this PR

**Review by JohanFreelancer9 — APPROVED** [review]

**Review by peterouma — COMMENTED** [review]
https://intranet.alxswe.com/corrections/10523815/correct

**Comment by scanum** [comment]
@MikePlumleyMSFT 
PR has been copyedited and is ready for final review, could you please check and merge? Thanks!

**Pull request merged**